LadderBots.json

```diff
@@ -2,14 +2,25 @@
 
 	"Bots": {
 		"CryptBot": {
-			"Race": "Terran",
+			"Race": "Protoss",
 			"Type": "BinaryCpp",
 			"Path": "bot_simple.exe"
 		},
-		"ProtossBot": {
+		"TerranBot": {
 			"Race": "Terran",
 			"Type": "BinaryCpp",
-			"Path": "bot_simple.exe"
+			"Path": "bot_simple.exe",
+			"Args": "-something"
+		},
+		"ZergBot": {
+			"Race": "Zerg",
+			"Type": "pythonSC2",
+			"Path": "bot_simple/run.py"
```

Comment:
Thats not exactly what was proposed because you don't split Path and Executable name.
It doesn't look like a problem anyway.

Reply:
Hm I mean in the end we would anyway then do Path + executable name. But maybe it is clearer what to write for the user... I will change it this evening.

Reply:
Your approach looks good to me too. Probably you should preserve it.
@Cryptyc ?

Reply:
Yeh, I'd definately be in favor of seperating them out, seems like a good way to go.  I can have a look at this later if you like.  I'm going to restructure everything so each bot is in a seperate subdirectory for the ladder anyway, to get rid of data directories, so this aproach would fit well with that